### PR TITLE
Misc changes 

### PR DESCRIPTION
**Pull request by andreitr**
- featured images 
- explore page copy
- linked listings on nft landing page

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -53,164 +53,140 @@ const metadata = {
 }
 const CAROUSEL = [
```
Comment (SentientClee) [inline pages/index.tsx:54]: Prolly a way to generate this dynamically in the future so we don't have to rely on code changes. But down the line forsure

**Review by SentientClee — APPROVED** [review]

File: pages/index.tsx [hunk 2]
```diff
@@ -53,164 +53,140 @@ const metadata = {
...
-      'https://i.seadn.io/gcs/files/6bfc31c8988a03c15a8858a94ed4da87.jpg?w=500&auto=format',
+    artist: 'niftyapes',
+    buyFinancingPrice: 0.825,
+    buyNowPrice: 3.35,
+    contractAddress: '0xa5ae59eee379fc02206d715b9431ffa53507c152',
+    image: 'http://localhost:3000/niftyapes/placeholder.png',
     rarity: '1/1',
-    title: 'Breaker',
-    tokenId: '275',
+    title: 'TEST TEST TEST',
```
Comment (SentientClee) [inline pages/index.tsx:188]: Is this cool to leave in?

Comment (andreitr) [reply pages/index.tsx:188]: Yes, I want to be obvious that we are on a testnet. We can replace this with featured goerli NFTs

File: pages/index.tsx (hunk 1 above)
Comment (andreitr) [reply pages/index.tsx:54]: Totally! We can get the listed NFTs and sort them by price or something.

**Pull request merged**